ide/che-core-ide-api/src/main/java/org/eclipse/che/ide/api/parts/PartStack.java

```diff
@@ -120,20 +120,20 @@
     void collapse();
 
     /**
-     * Minimizes / hides the part stack.
-     * The part state will not be retored when restoring the perspective state.
+     * Minimizes (hides) the part stack.
+     * The part state will not be restored when restoring the perspective state.
      */
     void minimize();
 
     /**
-     * Restores the part stack and the perspective to the default state.
+     * Restores the part stack from minimized state.
      */
-    void restore();
+    void unMinimize();
```

Comment:
A good question. I gave wrong names for methods.
I have to rename minimize() to hide() and unMinimize() to show().
Part can be collapsed when one other part is maximized. And then all collapsed parts must be restored when restoring maximized path.